src/Orleans.Runtime/Core/Dispatcher.cs

```diff
@@ -238,10 +238,10 @@ private void ReceiveResponse(Message message, ActivationData targetActivation)
         {
             lock (targetActivation)
             {
-                if (targetActivation.State == ActivationState.Invalid)
+                if (targetActivation.State == ActivationState.Invalid || targetActivation.State == ActivationState.FailedToActivate)
                 {
                     logger.Warn(ErrorCode.Dispatcher_Receive_InvalidActivation,
-                        "Response received for invalid activation {0}", message);
+                        "Response received for {state} activation {message}", targetActivation.State, message);
```

Comment:
I think we need to change this to `LogWarning((int)ErrorCode.`. Also, guidelines say to use PascalCase for the template parameter names